src/resources/js/views/tooltip.js

```diff
@@ -40,7 +40,7 @@ tribe.events.views.tooltip = {};
 	 */
 	obj.config = {
 		delayHoverIn: 600,
```

Comment:
i think we were reducing the hover in to 300. I'm not sure if we were reducing the hover out though.